src/main/java/com/redhat/cloud/notifications/events/EventConsumer.java

```diff
@@ -38,7 +39,7 @@ public EventConsumer(MeterRegistry registry) {
     }
 
     @Incoming("ingress")
-//    @Acknowledgment(Acknowledgment.Strategy.MANUAL)
+    @Acknowledgment(Strategy.PRE_PROCESSING)
     // Can be modified to use Multi<Message<String>> input also for more concurrency
     public Uni<Void> processAsync(Message<String> input) {
```

Comment:
Should we make this `void` instead of `Uni<Void>` now that we don't return anything anymore?

Reply:
Probably leave as is for now even if we could make the change